app/views/layouts/application.html.erb

```diff
@@ -9,48 +9,44 @@
   </head>
 
   <body>
-  <nav class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
-
-    <h1><a class="navbar-brand" href="#">Obituary Hub</a></h1>
-    <nav class="navbar-right navbar-light bg-light">
-      <form class="form-inline">
-        <% if current_user %>
-          <button class="btn btn-sm btn-outline-secondary" type="button">Logout</button>
-        <% else %>
-          <button class="btn btn-sm btn-outline-secondary" type="button">Login</button>
-          <button class="btn btn-sm btn-outline-secondary" type="button">Register</button>
-        <% end %>
-      </form>
-    </nav>
-  </nav>
-
-  <nav class="navbar navbar-expand-lg navbar-light bg-light style="background-color: #e3f2fd;">
-
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="navbarText">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item active">
-          <a class="nav-link" href="#">All Obituaries <span class="sr-only">(current)</span></a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">COVID-19 Obituaries</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">Recently Published</a>
-        </li>
-      </ul>
+    <nav1 class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
+      <h1><a class="navbar-brand" href="/">Obituary Hub</a></h1>
```

Comment:
Added all the links, including a link to redirect to the homepage from any other page when you click the app name

Reply:
Nice Work